src/main/scala/ScalaModulePlugin.scala

```diff
@@ -140,6 +143,10 @@ object ScalaModulePlugin extends AutoPlugin {
         </developer>
       </developers>
     )
+  )
+
+  lazy val scalaModuleSettingsJVM: Seq[Setting[_]] = Seq(
+    mimaPreviousVersion := None
   ) ++ mimaSettings ++ scalaModuleOsgiSettings
```

Comment:
Hum I just realized that this means you've deactivated MiMa for Scala.js, doesn't it?

Scala.js artifacts can very well be checked by MiMa, and should be. We have an (almost) guarantee that "JVM binary compatible" implies "SJSIR binary compatible". So if MiMa says OK for the .class files in a Scala.js artifact, it also means OK for the .sjsir files. It's very useful. We check our own artifacts with MiMa.